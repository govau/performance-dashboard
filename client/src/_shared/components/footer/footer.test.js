/* global it, describe */
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './footer';

describe('(Components) Footer - footer ', () => {
  it('should match Snapshot', () => {
    const renderedValue = renderer.create(<Footer />).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
