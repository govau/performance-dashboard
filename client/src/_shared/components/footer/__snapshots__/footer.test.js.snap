exports[`(Components) Footer - footer  should match Snapshot 1`] = `
<footer
  className="main__footer">
  <div
    className="govau-footer">
    <div
      className="container">
      <div
        className="govau-footer__container">
        <div
          className="govau-footer__container__logo">
          <img
            alt="Commonwealth of Australia Coat of Arms Logo"
            height="135"
            src="test-file-stub"
            width="185" />
        </div>
        <div
          className="govau-footer__container__utils">
          <div
            className="line-1">
            <a
              className="UIK-button btn btn-secondary report-an-issue"
              href="mailto:performance-dashboard@dta.gov.au"
              rel="external">
              Report an issue
            </a>
          </div>
          <div
            className="line-2">
            <small>
              Â© Commonwealth of Australia
            </small>
            <p
              className="small">
              This work is licensed under a Creative Commons Attribution 3.0 International
               
              <a
                className="UIK-link "
                href="https://creativecommons.org/licenses/by/3.0/au/deed.en"
                rel="external">
                License
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
